lambda-layer/src/Dockerfile

```diff
@@ -6,10 +6,13 @@ WORKDIR /workspace
 
 RUN sed -i "/opentelemetry-exporter-otlp-proto-grpc/d" ./aws-opentelemetry-distro/pyproject.toml
 
+RUN sed -i "/opentelemetry-instrumentation-aws-lambda/d" ./aws-opentelemetry-distro/pyproject.toml
+
 RUN sed -i "/opentelemetry-instrumentation-system-metrics/d" ./aws-opentelemetry-distro/pyproject.toml
 
 RUN mkdir -p /build && \
   python3 -m pip install aws-opentelemetry-distro/ -t /build/python && \
+  cp -r opentelemetry /build/python && \
```

Comment:
q: do you see the layer size changed?

Reply:
No, I move aws_lambda instrumentation from pipy dependency to local folder. The file size is same.